articles/api/management/v2/tokens.md

```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
 
 ## How to get a Management APIv2 Token
 
+### Using the Client Credentials Flow
+
+**Create a Non Interactive Client**
+
+A Non Interactive Client is a client that interacts with an API where there is no user involved. It's a machine to machine interaction. This must be used instead of a Single Page or Native apps because those cannot meet the necessary security requirements for executing this type of flow.
+
+To create a new Non Interactive Client, go to the [Clients section](${manage_url}/#/clients) of the dashboard then click the **CREATE CLIENT** button.
```

Comment:
In order for this to work the *Account Settings > Advanced > Enable APIs Section* has to be enabled. If it is not, then I do not see the *APIs* menu on the left and I do not have the *Non Interactive Client* option when I create a new Client. You can check [in this doc](https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0).